README.md

```diff
@@ -30,160 +30,496 @@ Those tags are generic enough to be used by other biomedical funders
+                           [MODEL_PATH]
+
+Arguments:
+  [DATA_PATH]             path to processed JSON data to be used for training
+  [LABEL_BINARIZER_PATH]  path to label binarizer
+  [MODEL_PATH]            path to output model.pkl or dir to save model
+
+Options:
+  --approach TEXT                 tfidf-svm, scibert, cnn, ...  [default:
+                                  tfidf-svm]
+
+  --parameters TEXT               model params in sklearn format e.g.
+                                  {'svm__kernel: linear'}
+
+  --test-data-path PATH           path to processed JSON test data
+  --threshold FLOAT               threshold to assign a tag
+  --data-format TEXT              format that will be used when loading the
+                                  data. One of list,generator  [default: list]
+
+  --test-size FLOAT               float or int indicating either percentage or
+                                  absolute number of test examples  [default:
+                                  0.25]
+
+  --sparse-labels / --no-sparse-labels
+                                  flat about whether labels should be sparse
+                                  when binarized  [default: False]
+
+  --cache-path PATH               path to cache data transformartions
+  --config PATH
+  --help                          Show this message and exit.
+```
+
+Label binarizer converts tags to a Y matrix with one column per tag. This is
+neccesary as this is a multilabel problem so multiple tags may be assigned to
+one grant.
+
+Parameters is a JSON like dump (essentially a stringified dict) with all params
+that the model expects in an sklearn fashion.
+
+### 📈 Evaluate
+
+Evaluate enables evaluation of the performance of various approaches including
+human performance and other systems like MTI, SciSpacy and soon Dimensions. As
+such evaluate has the followin subcommands
+
+#### Model
+
+Model is the generic entrypoint for model evaluation. Similar to train approach
+controls which model will be evaluated. Approach which is a positional argument
+in this command controls which model will be evaluated. Since the data in train
+are sometimes split inside train, the same splitting is performed in evaluate.
+Evaluate only supports some models, in particular those that have made it to
+production. These are: `tfidf-svm`, `scibert`, `science-ensemble`, `mesh-tfidf-svm`
+and `mesh-cnn`. Note that train also outputs evaluation scores so for models
+not made into production this is the way to evaluate. The plan is to extend 
+evaluate to all models when train starts training explicit model approaches. 
+
+```
+Usage: grants_tagger evaluate model [OPTIONS] APPROACH MODEL_PATH DATA_PATH
+                                    LABEL_BINARIZER_PATH
+
+Arguments:
+  APPROACH              model approach e.g.mesh-cnn  [required]
+  MODEL_PATH            comma separated paths to pretrained models  [required]
+  DATA_PATH             path to data that was used for training  [required]
```

Comment:
do you somehow ensure the same split seed is used for this function as it was in training?

Reply:
yes